components/virtual-node/virtual-node-write-service-impl/src/main/kotlin/net/corda/virtualnode/write/db/impl/writer/VirtualNodeEntityRepository.kt

```diff
@@ -155,7 +156,7 @@ internal class VirtualNodeEntityRepository(private val entityManagerFactory: Ent
 
     internal fun getVirtualNode(holdingIdentityShortHash: String): VirtualNodeInfo {
         entityManagerFactory.transaction {
-            val virtualNodeEntity = it.findVirtualNode(holdingIdentityShortHash)
+            val virtualNodeEntity = it.findVirtualNode(holdingIdentityShortHash.uppercase(Locale.getDefault()))
```

Comment:
```suggestion
            val virtualNodeEntity = it.findVirtualNode(holdingIdentityShortHash.uppercase())
```
Unless there is a specific reason for passing default Locale.